webpack_config/webpack.config.base.js

```diff
@@ -33,7 +45,11 @@ var config = {
             {
                 test: /fg-loadcss\/src\/onloadCSS/,
                 loader: 'exports?onloadCSS'
-            }
+            },
+            {
+                test: /\.html$/,
+                loader: 'vue-html'
```

Comment:
That's a good question – I copied this rule from the vue-cli scaffolding without fully understanding it. I'll look into what it's doing and remove it if it's not necessary.